src/Microsoft.Diagnostics.NETCore.Client/ReversedServer/ReversedDiagnosticsServer.cs

```diff
@@ -21,7 +21,7 @@ internal sealed class ReversedDiagnosticsServer : IAsyncDisposable
     {
         // The amount of time to allow parsing of the advertise data before cancelling. This allows the server to
         // remain responsive in case the advertise data is incomplete and the stream is not closed.
-        private static readonly TimeSpan ParseAdvertiseTimeout = TimeSpan.FromMilliseconds(250);
+        private static readonly TimeSpan ParseAdvertiseTimeout = TimeSpan.FromMilliseconds(750);
```

Comment:
I have no background information on why 250 ms was chosen as default, but if we want to keep the default, we could add a new constructor to ReversedDiagnosticsServer (it's an internal class) and then pass the timeout when constructing the instance (WebSocketServerRouterFactory) or we could even make it configurable as a passed in argument when running commands that ends up constructing a ReversedDiagnosticsServer in dsrouter.